client/src/main/java/com/sap/mobile/services/client/push/BaiduNotification.java

```diff
@@ -7,15 +7,17 @@
 import lombok.RequiredArgsConstructor;
 
 public interface BaiduNotification {
-	
+
 	static Builder builder() {
 		return new Builder();
 	}
-	
+
 	BaiduNotificationAndroid getAndroid();
-	
+
 	BaiduNotificationIos getIos();
-	
+
+	Integer getMsgType();
```

Comment:
Maybe we can offer an enum and map it to Integer in our internal representation sent to the server?

Reply:
Makes sense